.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,67 @@
+name: "\U0001F41E Report a problem"
+description: "Report an issue with ESLint or rules bundled with ESLint"
+title: "Bug: (fill in)"
+labels:
+  - bug
+  - "repro:needed"
+body:
+- type: markdown
+  attributes:
+    value: ESLint adheres to the [Open JS Foundation Code of Conduct](https://eslint.org/conduct). By opening an issue, you agree to abide by the code of conduct.
+- type: textarea
+  attributes:
+    label: Environment
+    description: |
+      Please tell us about how you're running ESLint (Run `npx eslint --env-info`.)
+    value: |
+        Node version: 
+        npm version: 
+        Local ESLint version: 
+        Global ESLint version: 
+        Operating System:
```

Comment:
Not needed for this PR, but should we update `eslint --env-info` to print OS or trust users to fill it out manually? Right now it only prints the first four versions.

Reply:
That was updated in https://github.com/eslint/eslint/pull/14059, if it doesn't appear in the output maybe we have a bug.

Reply:
Aha, I happened to run this from a repository with a slightly older version of ESLint. All good on recent versions!